src/org/ods/quickstarter/Context.groovy

```diff
@@ -0,0 +1,79 @@
+package org.ods.quickstarter
+
+import com.cloudbees.groovy.cps.NonCPS
+
+class Context implements IContext {
+
+  private Map config
+
+  Context(Map config) {
+    this.config = config
+  }
+
+  String getJobName() {
+    config.jobName
+  }
+
+  String getBuildNumber() {
+    config.buildNumber
+  }
+
+  String getBuildUrl() {
+    config.buildUrl
+  }
+
+  String getBuildTime() {
+    config.buildTime
+  }
+
+  @NonCPS
```

Comment:
Is there a reason why this is only on some methods?

Reply:
I forgot them. Added in 4df3615.